doc/admin/installation.md

```diff
@@ -38,7 +38,7 @@ repositories.
 
 Add the DAOS package repository via zypper:
 ``` bash
-$ sudo zypper ar http://packages.daos.io/v1.2/Leap15.2/packages x86_64/daos_packages
+$ sudo zypper ar http://packages.daos.io/v1.2/Leap15/packages x86_64/daos_packages
```

Comment:
zypper ar https://packages.daos.io/v1.2/Leap15/packages/x86_64/ daos_packages

Note the URL always has a trailing slash, and the last parameter is a user alias that does not have a slash.